### PR TITLE
Bank account by Kate

### PR DESCRIPTION
**Pull request by ekaterinakur**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+class Person {
+  constructor(name, birthday, amount) {
+    this.name = name;
+    this.birthday = birthday;
+    this.initialAmount = amount;
+    this.history = [`Initial: ${this.initialAmount}`];
```
Comment (gestap) [inline src/main.js]: will be better if you saved transaction in `object` instead of `string`

File: src/main.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  }
+
+  getAge() {
+    const currentDate = new Date();
+    const birthDate = this.birthday.split('.');
+    const months = currentDate.getMonth() - (birthDate[1] - 1);
```
Comment (gestap) [inline src/main.js]: it's hard to understand what in `birthDate[n]`

File: src/main.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    return age;
+  };
+
+  getInfo() {
+    console.log(`Name: ${this.name}, Age: ${this.getAge()}, Amount: ${this.initialAmount}$`);
+  }
+
+  addMoney(value, from) {
+    this.initialAmount += value;
+    this.history.push(`${from}: ${value}`);
```
Comment (gestap) [inline src/main.js]: create a function for this operation

File: src/main.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  }
+
+  withdrawMoney(value, asgmt) {
+    this.initialAmount -= value;
+    this.history.push(`${asgmt}: -${value}`);
+  }
+
+  getAccountHistory() {
+    console.log(this.history);
```
Comment (gestap) [inline src/main.js]: change to `return`

File: src/main.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    
+    if (months < 0 || (months === 0 && currentDate.getDate() < birthDate[0])) {
+      age--;
+    }
+
+    return age;
+  };
+
+  getInfo() {
+    console.log(`Name: ${this.name}, Age: ${this.getAge()}, Amount: ${this.initialAmount}$`);
```
Comment (gestap) [inline src/main.js]: change to `return`

File: src/main.js (hunk 1 above)
Comment (ekaterinakur) [reply src/main.js]: Done

File: src/main.js (hunk 2 above)
Comment (ekaterinakur) [reply src/main.js]: Done

File: src/main.js (hunk 3 above)
Comment (ekaterinakur) [reply src/main.js]: Done, if I understood correctly

File: src/main.js (hunk 4 above)
Comment (ekaterinakur) [reply src/main.js]: Done

File: src/main.js (hunk 5 above)
Comment (ekaterinakur) [reply src/main.js]: Done

**Pull request closed without merging**